IBX-1440: Add description metadata (#133)
ezsystems/user-documentation: docs/personalization/event_types.md

---

> description: Recommendations rely on tracking different events describing the behavior of users on your website.

Author: ```suggestion
description: Recommendations rely on tracking different events that describe users' behavior on the website.
```
